<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QWidget" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>761</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label_subdirectories">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>651</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Please input Sub-directories (comma separated, no SPACE after comma)</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_directory">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>541</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Please input or select directory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_directory">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>~/diffusion/</string>
     </property>
     <property name="placeholderText">
      <string>eg: ~/diffusion/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_scatter">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>150</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot
 Im vs Re
 scatters</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_directory">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>80</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>select directory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_subdirectories">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>00004,00007,00013,00022,00044,00073</string>
     </property>
     <property name="placeholderText">
      <string>eg: 00004,00007,00013,00022,00044,00073</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_response">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>~/simulation_Im_Re_curve/diffTip.csv</string>
     </property>
     <property name="placeholderText">
      <string>eg: ~/simulation_Im_Re_curve/diffTip.csv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_response">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>250</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>select csv file</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="Widget_response" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>411</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_draw">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>380</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_im">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>Ch1 retrace.txt</string>
     </property>
     <property name="placeholderText">
      <string>eg: Ch1 retrace.txt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_re">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Ch2  retrace.txt</string>
     </property>
     <property name="placeholderText">
      <string>eg: Ch2 retrace.txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_im">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MIM-Im file name:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_re">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MIM-Re file name:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_image_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../qttest4/img.jpg</pixmap>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_image_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../qttest4/img.jpg</pixmap>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../qttest4/img.jpg</pixmap>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_center">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_image_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../qttest4/img.jpg</pixmap>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>240</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_response">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>250</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
